src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -378,6 +380,14 @@ private string GetSourceLink(Uno.Roslyn.MSBuildItem projectItemInstance)
 			}
 		}
 
+		private void TryGenerateUnoResourcesKeyAttribute(ImmutableHashSet<string> resourceKeys)
+		{
+			if (resourceKeys.IsEmpty)
+			{
+				_generatorContext.AddSource("LocalizationResources", "[assembly: global::System.Reflection.AssemblyMetadata(\"UnoHasLocalizationResources\", \"True\")]");
+			}
```

Comment:
It seems counter-intuitive that having now resources (is empty) is setting a flag telling there is localized resources...